eval/testdata/complex.ng

```diff
@@ -1,5 +1,21 @@
+complex(1, 2)
+complex(1, 2.2)
+complex(1, float32(2))
+complex(1, float64(2))
+
+complex(1.1, 2)
+complex(1.1, 2.2)
+complex(1.1, float32(2))
+complex(1.1, float64(2))
+
+complex(float32(2), float32(3))
+
+complex(float64(2), float64(3))
+
 // TODO: c := complex128(0) + complex128(complex64(3.5))
 // TODO: real(1 + 2i) == 1.0
```

Comment:
Looks like these will work now. Want to remove the comments and replace the print at the bottom with something like:

if (some addition of complex numbers) == (some converted value) {
    print("OK")
}

Reply:
unfortunately, no.

```
ng> complex(1,2)
complex128((1+2i))

ng> c := complex(1,2)
ng eval panic: TODO rtype for Complex
goroutine 1 [running]:
runtime/debug.Stack(0x26, 0x0, 0x0)
	/usr/lib/go/src/runtime/debug/stack.go:24 +0xa7
runtime/debug.PrintStack()
	/usr/lib/go/src/runtime/debug/stack.go:16 +0x22
neugram.io/ng/eval.(*Program).Eval.func1(0xc420156090, 0xc42010b718, 0xc42010b700)
	/home/binet/work/gonum/src/neugram.io/ng/eval/eval.go:411 +0x204
panic(0x8dfa00, 0xa138c0)
	/usr/lib/go/src/runtime/panic.go:491 +0x283
neugram.io/ng/eval.(*reflector).ToRType(0xc420148040, 0xd81d00, 0xa13cf0, 0xc4200f2240, 0x1)
	/home/binet/work/gonum/src/neugram.io/ng/eval/eval.go:1423 +0x265c
neugram.io/ng/eval.(*Program).evalStmt(0xc420156090, 0xd80cc0, 0xc4200f4780, 0x0, 0x0, 0x0)
	/home/binet/work/gonum/src/neugram.io/ng/eval/eval.go:466 +0x18cf
neugram.io/ng/eval.(*Program).Eval(0xc420156090, 0xd80cc0, 0xc4200f4780, 0xc42005e5a0, 0x0, 0x0, 0x0, 0xd80380, 0xc420148200)
	/home/binet/work/gonum/src/neugram.io/ng/eval/eval.go:424 +0x1fc
main.handleResult(0x1, 0xc4201481b0, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/binet/work/gonum/src/neugram.io/ng/ng.go:338 +0x734
main.loop()
	/home/binet/work/gonum/src/neugram.io/ng/ng.go:331 +0x602
main.main()
	/home/binet/work/gonum/src/neugram.io/ng/ng.go:119 +0x35c
ng: ng eval panic: TODO rtype for Complex

ng> 
```